tests/lib/rules/define-props-declaration.js

```diff
@@ -108,6 +111,7 @@ tester.run('define-props-declaration', rule, {
     }
   ],
   invalid: [
+    // default
```

Comment:
Could you please add a test case to check if an error occurs when an array is used for props?

For example: 

```vue
      <script setup lang="ts">
      const props = defineProps(['kind'])
      </script>
```

Reply:
I've implemented handling of such case - the kind is a not required of type string.